packages/vxrn/src/exports/prebuild.ts

```diff
@@ -3,60 +3,183 @@ import path from 'node:path'
+    }
+    const srcPermissions = FSExtra.statSync(srcPath).mode
+    let content = FSExtra.readFileSync(srcPath, 'utf8')
+    Object.entries(replacements).forEach(([regex, value]) => {
+      content = content.replace(new RegExp(regex, 'g'), value)
+    })
+    FSExtra.writeFileSync(destPath, content, {
+      encoding: 'utf8',
+      mode: srcPermissions,
+    })
+  })
+}
+/*
+End of meta owned code
+End of copyright Facebook
+*/
+
+const transformPath = (filePath: string) => {
+  return filePath
+    .replace('_BUCK', 'BUCK')
+    .replace('_gitignore', '.gitignore')
+    .replace('_gitattributes', '.gitattributes')
+    .replace('_babelrc', '.babelrc')
+    .replace('_editorconfig', '.editorconfig')
+    .replace('_eslintrc.js', '.eslintrc.js')
+    .replace('_flowconfig', '.flowconfig')
+    .replace('_buckconfig', '.buckconfig')
+    .replace('_prettierrc.js', '.prettierrc.js')
+    .replace('_bundle', '.bundle')
+    .replace('_ruby-version', '.ruby-version')
+    .replace('_node-version', '.node-version')
+    .replace('_watchmanconfig', '.watchmanconfig')
+    .replace('_xcode.env', '.xcode.env')
+}
 
 export const prebuild = async ({
   root,
   platform,
-}: { root: string; platform?: 'ios' | 'android' | string }) => {
+  expo = true,
+}: { root: string; platform?: 'ios' | 'android' | string; expo: boolean }) => {
   const options = await fillOptions({ root })
 
   applyBuiltInPatches(options).catch((err) => {
     console.error(`\n 🥺 error applying built-in patches`, err)
   })
+  const doesIOSExist = FSExtra.existsSync(path.resolve('ios'))
+  const doesAndroidExist = FSExtra.existsSync(path.resolve('android'))
 
-  try {
-    // Import Expo from the user's project instead of from where vxrn is installed, since vxrn may be installed globally or at the root workspace.
-    const require = module.createRequire(root)
-    const importPath = require.resolve('@expo/cli/build/src/prebuild/index.js', {
-      paths: [root],
-    })
-    const expoPrebuild = (await import(importPath)).default.expoPrebuild
-    await expoPrebuild([
-      ...(platform ? ['--platform', platform] : []),
-      '--skip-dependency-update',
-      'react,react-native,expo',
-    ])
+  if (expo) {
+    try {
+      // Import Expo from the user's project instead of from where vxrn is installed, since vxrn may be installed globally or at the root workspace.
+      const importPath = require.resolve('@expo/cli/build/src/prebuild/index.js', {
+        paths: [root],
+      })
+      const expoPrebuild = (await import(importPath)).default.expoPrebuild
+      await expoPrebuild([
+        ...(platform ? ['--platform', platform] : []),
+        '--skip-dependency-update',
+        'react,react-native,expo',
+      ])
 
+      try {
+        const packageJsonPath = path.join(root, 'package.json')
+        let packageJsonContents = await FSExtra.readFile(packageJsonPath, 'utf8')
+
+        packageJsonContents = packageJsonContents.replace(/expo run:ios/g, 'one run:ios')
```

Comment:
probably don't want to do this, or maybe just print warnings with an option to `--fix`

Reply:
fixed